CMakeLists.txt

```diff
@@ -294,8 +294,8 @@ else()
   add_definitions(-DMXNET_USE_NCCL=0)
 endif()
 
+include(cmake/ChooseBlas.cmake)
```

Comment:
Lots of stuff from here will be overwritten (and misconfigured) due to the `else()` branch, when the [mshadow scripts](https://github.com/dmlc/mshadow/blob/master/cmake/mshadow.cmake) kick in. On the other hand if it passes CI, then why not?

Reply:
Well, I need the defines for the feature detection basically.

Reply:
Yes, as @lebeg mentioned this change will do nothing in the current state of the code, so this PR doesn't bring anything to the table.

I am currently working on making linking with BLAS a little bit easier, specially on Windows.

Reply:
It works for the feature detection though